mung_manager/commons/constants.py

```diff
@@ -68,6 +68,10 @@ class SYSTEM_CODE:
         "already_exists_user_pet_kindergarden",
         "User already has a pet kindergarden.",
     )
+    INVALID_PET_KINDERGARDEN_ID = (
+        "invalid_pet_kindergarden_id",
+        "The user does not belong to that pet kindergarden.",
+    )
```

Comment:
`INVALID`보다는 `NOT_FOUND`가 더 좋지않을까 생각이듭니다. 

아래처럼 수정해보면 어떨까요? 제생각에 아래 에러가 있는 것으로 알고 있습니다 :)

```suggestion
    NOT_FOUND_PET_KINDERGARDEN = (
        "not_found_pet_kindergarden",
        "Pet kindergarden does not exist.",
    )
```

에러 메시지 작성에 대해서 규칙이 모호하지만 나름 규칙을 정하는 팁이 있습니다.

클라이언트에서 알 수는 있지만 그 에러 상황에 대해서 정확한 에러를 알면 안된다고 생각이 듭니다.  중간선을 잘 아시는게 중요할 것 같습니다.